apps/api/src/app/organization/usecases/rename-organization/rename-organization-command.ts

```diff
@@ -0,0 +1,10 @@
+import { IsDefined, MaxLength } from 'class-validator';
+import { AuthenticatedCommand } from '../../../shared/commands/authenticated.command';
+
+export class RenameOrganizationCommand extends AuthenticatedCommand {
+  @IsDefined()
+  public readonly id: string;
+
+  @MaxLength(50)
```

Comment:
Just noticed this. I think it is better to not limit the max length because in the creation of organization we don't limit the length and it would be a misalignment. It is a good thing to have done that, but for now we can skip it until we commit to a decision to limit org name and implement it everywhere, also at database level. 👍

Reply:
I tend to aggree that it should be better implemented across the board, was initially thinking we can only implement it here. Let me revert that rule